blog/2020-05-01-april-dev-update.md

```diff
@@ -0,0 +1,63 @@
+---
+
+> It’s difficult to make predictions, especially about the future.
+>
+> Niels Bohr
+
+Are we on the verge of plummeting into the biggest recession since the big crash of 1929? 
+And if so, are cryptocurrencies the answer, a safe haven to put your money in?
+
+While we cannot look into the future and don't dare to predict it, we are inclined to believe that we are *likely* to see a high demand for crypto in the future, and an even higher one in the next few months. 
+Time to get COMIT out there so you don't have to trust those central exchanges, but keep your private keys where they should be: in your wallet.
+Verify, don't trust!
+
+Join us for a discussion around advances in crypto, trustless trading and - as usual - various tech related topics in [next week Thursday's COMIT Community ☕️](https://calendar.google.com/event?action=TEMPLATE&tmeid=NHEzb2tkNDJtcXFzdnNyNG9oM2UzNGhpc3MgdGVueC50ZWNoX202dWp2OXQ4cGluajAxYmI2NXV0aHRmcnM0QGc&tmsrc=tenx.tech_m6ujv9t8pinj01bb65uthtfrs4%40group.calendar.google.com) - 7pm Sydney time!
+
+<!--truncate-->
+
+It would be an understatement to say that the COMIT team is quite tech-focused.
+We breath tech, usually focusing on discussions around protocol implementations rather than philosophising about the global financial system.
+However, with the recent turmoil caused by COVID-19, we had a noticeable trend towards discussing topics outside the tech sphere.
+
+While some plans, like filling up your bathtub with crude oil and getting paid for it, were fun but not the most practical, other questions, like should I invest now? - and if yes into what? - were interesting to the team.
+And we don't seem to be the only ones.
+We are used to the cryptocurrency market being volatile - but recently the traditional stock market has been even less predicatable.
+Making quick money might, however, be a shallow tactic, given that we might just be at the start of the biggest bear market to date.
+By looking at [statistics of previous historical crashes](https://todaytrader.com/blog/stock-market-crash-preparation/) it becomes clear that we might not have reached the bottom yet, and the time to break-even could be decades away.
+We love statistics, but as the quote at the top of this post says - predicting the future is a bet.
+It can be an educated bet, but nonetheless a bet.
+In the end it's up to each individual to come to a conclusion on how to handle this financial crisis.
+Let's end this excursion with a second quote I really like: 
+
+> 49% losses, 51% wins, and lived well from the difference.
+>
+> André Kostolany
+
+Back to the tech - let's have a look on what happened in COMIT last month!
+With the beginning of the new quarter we started tackling new challenges while still finishing some issues from the previous quarter.
+
+In the first week of April the Lightning squad - Thomas, Franck and Tobin - [successfully integrated Lightning-ERC20 swaps](https://github.com/comit-network/comit-rs/pull/2426) into cnd.
```

Comment:
Afaik this was HanETH-HaLight
```suggestion
In the first week of April the Lightning squad - Thomas, Franck and Tobin - [successfully integrated Ether-Lighting swaps](https://github.com/comit-network/comit-rs/pull/2426) into cnd.
```

Reply:
oh, good catch, it's Ether - what was I thinking :D